.github/workflows/publish-container.yaml

```diff
@@ -0,0 +1,120 @@
+name: '🚀 Publish Container Images'
+
+on:
+  push:
+    branches:
+      - master
+  release:
+    types:
+      - published
+  workflow_dispatch:
+
+jobs:
+  setup-vars:
+    name: Setup Variables
+    runs-on: ubuntu-latest
+    if: ${{ github.repository == 'jellyfin/jellyfin-web' }}
+
+    outputs:
+      stage: ${{ steps.variables.outputs.stage }}
+      short-sha: ${{ steps.variables.outputs.short-sha }}
+      major-minor: ${{ steps.variables.outputs.major-minor }}
+      version: ${{ steps.variables.outputs.version }}
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Setup YQ
+        uses: chrisdickinson/setup-yq@latest
+        with:
+          yq-version: v4.9.8
+
+      - name: Setup sariables
+        id: variables
+        run: |-
+          if [[ "${GITHUB_REF}" =~ '^refs/tags/v.*$' ]]; then
+            STAGE='stable'
+          fi
+          VERSION=`yq e '.version' build.yaml`
+          echo "::set-output name=stage::${STAGE:-unstable}"
+          echo "::set-output name=short-sha::${GITHUB_SHA::7}"
+          echo "::set-output name=major-minor::${VERSION%.*}"
+          echo "::set-output name=version::${VERSION}"
+
+  container-image:
+    name: Build Container Images
+    runs-on: ubuntu-latest
+    needs:
+      - setup-vars
+
+    env:
+      STAGE: ${{ needs.setup-vars.outputs.stage }}
+      SHORT_SHA: ${{ needs.setup-vars.outputs.short-sha }}
+      MAJOR_MINOR: ${{ needs.setup-vars.outputs.major-minor }}
+      SEMVER: ${{ needs.setup-vars.outputs.version }}
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Setup Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+          check-latest: true
+          cache: 'npm'
+
+      - name: Install Dependencies
+        run: npm ci --no-audit
+        env:
+          SKIP_PREPARE: true
+
+      - name: Build Web
+        run: |-
+          npm run build:production
+          mv dist deployment/dist
+
+      - name: Build standalone web image
```

Comment:
Same as I mentioned in the metapackages PR, it seems our Vue actions are easier to use.

Reference: https://github.com/jellyfin/jellyfin-vue/blob/master/.github/workflows/docker-release.yml#L30

Reply:
Unelss you make it a blocking mater I will not change them sorry. I have tested all of this and changing parts out is not on my todo list atm (+ docker does not build OCI images this does)

Reply:
No worries. Just was curious about the choice and thought it had a *deeper* reason, but it's fine. Basically because if you're happy with it, we might want to switch to these actions as well in Vue :).

You're the one doing this so you're free to use whatever you feel like btw!